web/src/features/moreCast2/components/ActualCell.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { TextField, Tooltip } from '@mui/material'
+import { theme } from 'app/theme'
+import { GridRenderCellParams } from '@mui/x-data-grid-pro'
+
+interface ActualCellProps {
+  missingActual: boolean
+  value: Pick<GridRenderCellParams, 'formattedValue'>
+}
+
+const MISSING_ACTUAL_MESSAGE = 'Observation not available from WF1.'
```

Comment:
Should we use 'actual' instead of 'observation' since the column is called actual?

Reply:
Done: https://github.com/bcgov/wps/pull/4045/commits/97e97efaa95be92f27ddcc7144d12400c1949200